usb/esp_tinyusb/include/tinyusb_net.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <stdint.h>
+#include "tinyusb_types.h"
+#include "esp_err.h"
+#include "sdkconfig.h"
+
+#if (CONFIG_TINYUSB_NET_MODE_NONE != 1)
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef void (*net_recv_handler_t)(void *buffer, uint16_t len);
+
+/**
+ * @brief ESP TinyUSB NCM driver configuration structure
+ */
+typedef struct {
+    const uint8_t *mac_addr; /*!< MAC address. Must be 6 bytes long. */
```

Comment:
Could we use an array instead of pointer?
```suggestion
    const uint8_t mac_addr[6];
```